docs/reference/function-configuration/batching.md

```diff
@@ -0,0 +1,53 @@
+# Event batching
+
+(!) Feature is supported only for:
```

Comment:
```suggestion
:bangbang: Feature is supported only for:
```
Nice to know: https://gist.github.com/rxaviers/7360908

Reply:
Actually, this translates to RST right?
So maybe https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#warnings ?

Reply:
@TomerShor no, that's .md file